cmd/build.go

```diff
@@ -0,0 +1,125 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/boson-project/faas"
+	"github.com/boson-project/faas/buildpacks"
+	"github.com/boson-project/faas/docker"
+	"github.com/ory/viper"
+	"github.com/spf13/cobra"
+)
+
+var buildCmd = &cobra.Command{
+	Use:        "build",
+	Short:      "Build an existing function project as an OCI image",
+	SuggestFor: []string{"biuld", "buidl", "built", "image"},
```

Comment:
AH, adding common misspelling is handy.  It enables sloppy scripting, but probably worth it to the interactive users!